src/Sylius/Bundle/AdminBundle/Resources/views/Order/Label/ShippingState/partially_shipped.html.twig

```diff
@@ -0,0 +1,4 @@
+<span class="ui blue{% if attached is defined and attached == true %} top attached{% endif %} label">
```

Comment:
`ready` state is blue, maybe different colour?

Reply:
No idea though, partially shipped is something between ready & shipped, blue & green, clock & plane; the most important thing here is not to crash admin interface, I'm open for colour / icon suggestions though :)

Reply:
Like in payments, partially paid has the same colour as cancelled one.